src/zope/app/apidoc/typemodule/type.py

```diff
@@ -69,10 +69,11 @@ def items(self):
         """See zope.container.interfaces.IReadContainer"""
         results = [(name, LocationProxy(iface, self, name))
                    for name, iface in getUtilitiesFor(self.interface)]
-        results.sort(lambda x, y: cmp(x[1].getName(), y[1].getName()))
+        results.sort(key=lambda x: x[1].__name__)
```

Comment:
Okay, what's a `Foo instance` doing in a variable called `iface`?  Copy pasta from code below that does `for name, iface in getUtilitiesFor(IInterface)` (and then sorts by `iface.getName()`)?

I suggest renaming `iface` to `util` and changing the sort key to `x[0]`.